src/ContainerLogs.jsx

```diff
@@ -51,7 +53,8 @@ class ContainerLogs extends React.Component {
             disableStdin: true,
             fontSize: 12,
             fontFamily: 'Menlo, Monaco, Consolas, monospace',
-            screenReaderMode: true
+            screenReaderMode: true,
+            scrollback: LOGS_MAX_SIZE,
```

Comment:
Is that needed to be set? You could have longer logs when you keep as the logs keep streaming in